polaris-tokens/src/token-groups/legacy.ts

```diff
@@ -3,17 +3,14 @@ export const legacy = {
     value: '514',
   },
   'choice-size': {
-    value: '20px',
+    value: '18px',
```

Comment:
I would take this opportunity to remove the legacy variable so we can remove them in the next breaking change.

Reply:
Where would it make sense for this one to live? It's used across a couple of different components so making it a locally scoped custom property doesn't seem right.

Reply:
Hard code it. Polaris coverage will pick it up. I know we are worried about keeping the value linked but I am unsure the value.

Reply:
After some back and forth we didn't end up changing this value so going to leave as is.